Makefile.bootfiles

```diff
@@ -0,0 +1,110 @@
+BASE:=base.tar.gz
+DEV_BUILD:=0
+
+DELTA_TARGET=out/delta.tar.gz
+
+ifeq "$(DEV_BUILD)" "1"
+DELTA_TARGET=out/delta-dev.tar.gz
+endif
+
+ifeq "$(SNP_BUILD)" "1"
+DELTA_TARGET=out/delta-snp.tar.gz
+endif
+
+SRCROOT=$(dir $(abspath $(firstword $(MAKEFILE_LIST))))
+
+PATH_PREFIX:=
+# These have PATH_PREFIX prepended to obtain the full path in recipies e.g. $(PATH_PREFIX)/$(VMGS_TOOL)
+VMGS_TOOL:=
+IGVM_TOOL:=
+KERNEL_PATH:=
+TAR2EXT4_TOOL:=bin/cmd/tar2ext4
```

Comment:
Should we add a make command to build this?

Reply:
I wanted to make sure that `Makefile.bootfiles` doesn't depend on go runtime. It's useful in a multi job build process, where different container images could be used. e.g. for creating `delta*.tar.gz` files it could be a `golang` based container, for creating SNP boot files it'll be a `python` based one.